Dockerfile

```diff
@@ -15,8 +15,14 @@ COPY charts /charts
 COPY --from=builder /go/bin/gardener-extension-provider-alicloud /gardener-extension-provider-alicloud
 ENTRYPOINT ["/gardener-extension-provider-alicloud"]
 
+############# gardener-extension-admission-alicloud
+FROM base as gardener-extension-admission-alicloud
+
+COPY --from=builder /go/bin/gardener-extension-admission-alicloud /gardener-extension-admission-alicloud
+ENTRYPOINT ["/gardener-extension-admission-alicloud"]
+
 ############# gardener-extension-validator-alicloud
 FROM base AS gardener-extension-validator-alicloud
```

Comment:
I see you renaming the `validator` to `admission`, why here still need to build `validator` image?

Reply:
It is required to make the build for this PR happy. For more details see https://github.com/gardener/gardener-extension-provider-gcp/pull/69/files#r421264085. After PR, a follow-up cleanup PR such as https://github.com/gardener/gardener-extension-provider-aws/pull/272/files is required.